code/src/java/pcgen/cdom/reference/CDOMFactory.java

```diff
@@ -142,7 +160,24 @@ public boolean populate(ReferenceManufacturer<T> parentCrm,
 	@Override
 	public ManufacturableFactory<T> getParent()
 	{
+		/*
+		 * CONSIDER This is a limitation that prevents this from being used for
+		 * AbilityCategory. Need to figure out if this should really know it's parent and
+		 * if so, how? (when the problem is that the parentage is really now in
+		 * ClassIdentity)
```

Comment:
Could inheritance via traits (interfaces with default methods) help?

Reply:
Maybe.  It's specifically CONSIDER not TODO since I want to remind myself of the scenario as I go about polishing ClassIdentity, Manufacturers, etc.  Things are likely to move around a bit and I don't want to actually try to solve it now because this is trying to be as small a PR as possible, knowing more ClassIdentity work is coming.